src/basic/lines.js

```diff
@@ -14,19 +14,30 @@
 	EXTERNAL DEPENDENCIES
  */
 
-import {curry, compose, not, equals, identity, cond} from 'ramda';
+const ramda = require('ramda');
```

Comment:
umm, so:

* tangram.cartodb references to tangram-carto/dist/bundle.js
* At tangram-carto level there is a process watching the changes and rebuilding the bundle.
* tangram.cartodb has another process watching for changes, including node_modules/tangram-carto/dist/bundle.js right?

My goal was being able to have only one watch at the top level watching all the dependency tree. In fact the best scenario would be enabling es6-modules at the top level so we can allow es6-modules in the dependency tree modules.